capa/v29.2.0/README.md

```diff
@@ -10,6 +10,23 @@
 
 ### cluster-aws [v2.0.0...v2.2.0](https://github.com/giantswarm/cluster-aws/compare/v2.0.0...v2.2.0)
 
+#### ⚠️ Breaking change
+
+- Do not allow additional properties in the following fields in order to avoid unnoticed typos:
+  - global.connectivity.network
+  - global.connectivity.network.pods
+  - global.connectivity.network.services
+  - global.connectivity.subnets[]
+  - global.connectivity.topology
+  - global.controlPlane
+  - global.controlPlane.additionalSecurityGroups[]
+  - global.controlPlane.machineHealthCheck
+  - global.controlPlane.oidc
+  - global.providerSpecific
+  - global.providerSpecific.instanceMetadataOptions
```

Comment:
```suggestion
- Do not allow additional properties in the following fields in order to avoid unnoticed typos:

  - `global.connectivity.network`
  - `global.connectivity.network.pods`
  - `global.connectivity.network.services`
  - `global.connectivity.subnets[]`
  - `global.connectivity.topology`
  - `global.controlPlane`
  - `global.controlPlane.additionalSecurityGroups[]`
  - `global.controlPlane.machineHealthCheck`
  - `global.controlPlane.oidc`
  - `global.providerSpecific`
  - `global.providerSpecific.instanceMetadataOptions`
```